app.py

```diff
@@ -205,19 +220,20 @@ def create_inspection():  # pragma: no cover
 @swag_from("docs/swagger/update_inspection.yaml")
 def update_inspection(inspection_id):  # pragma: no cover
     try:
+        # Get JSON form from the request
+        inspection = request.get_json()
+        if inspection is None:
+            return jsonify(
+                error="Missing fertiliser form!"
+            ), HTTPStatus.BAD_REQUEST
         with connect_db(FERTISCAN_DB_URL, FERTISCAN_SCHEMA) as conn:
             with conn.cursor() as cursor:
+                create_search_path(conn, cursor, FERTISCAN_SCHEMA)
```

Comment:
Try without `create_search_path(conn, cursor, FERTISCAN_SCHEMA)`. Because when using `connect_db`, the path is already set to `FERTISCAN_SCHEMA` but also to `public` where some of the functions `update_inspection` relies on are located. Specifically setting `create_search_path(conn, cursor, FERTISCAN_SCHEMA)` removes `public`.

Reply:
Using `create_search_path(conn, cursor, FERTISCAN_SCHEMA)` was a recommendation of @Francois-Werbrouck . I'll try to remove it and see how it goes.